app/javascript/components/common/SearchInput.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useRef, useState } from 'react'
+import { GraphicalIcon } from './GraphicalIcon'
+
+type SearchInputProps = {
+  filter: string | undefined
+  setFilter: (filter: string) => void
+  placeholder: string
+  className?: string
+}
+
+const WRAPPER_CLASSNAMES =
+  'bg-unnamed15 text-textColor6 flex flex-row flex-grow rounded-[5px] border-1 border-transparent py-[11px] px-[21px] text-16 max-w-[420px] focus-within:focused-input hover:cursor-text'
```

Comment:
If there's a px padding of 21px, that's Tai making a mistake, so round it to 20px (I appreciate you doing it pixel perfect as a mindset though!)

For the py-11px, I'm presuming this is because the search box wants to be a fixed height. I'd rather specify the height than an arbitary vertical padding for that. 

```suggestion
  'bg-unnamed15 text-textColor6 flex flex-row flex-grow rounded-[5px] border-1 border-transparent py-[11px] px-20 text-16 max-w-[420px] focus-within:focused-input hover:cursor-text'
```

There are already a lot of other search boxes around the site. Are they different to this, or should they be standardised? If so, should we be reusing some of the CSS from them? (Genuine questions - I don't remember 🙂 )